### DISCUSSION TITLE
Windows Unpackaged

### DISCUSSION THREAD

**Discussion by tomasthedev** [unoplatform/uno#13267] [Q&A] [answered]
Hello, I facing issue with running Windows (Unpackaged) x86 and x64, when I try run that it will raise expection "System.DllNotFoundException: 'Unable to load DLL 'Microsoft.ui.xaml.dll' or one of its dependencies: Specified module wasn't located. (0x8007007E)'". I tried do everything what I found but witout succes. Someone suggest to replace something in launchsettings.json but I have exact like him. And the ui.xaml is installed with nuget.
Anyone know what to do?
Thanks

**Comment by jeromelaban** [comment]
Thanks for the question. Take a look at the [documentation on this topic](https://platform.uno/docs/articles/features/winapp-sdk-specifics.html).

**Reply by tomasthedev** [reply, marked as answer]
Thanks, I tried open windows .csproj and I was need uncomment line `<WindowsPackageType>None</WindowsPackageType>`
then it worked.
